src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ImportStatusBauService.cs

```diff
@@ -48,11 +49,30 @@ public class ImportStatusBauService : IImportStatusBauService
 
                     return queryResults.Where(result =>
                     {
-                        if (result.Message == "")
+                        try
+                        {
+                            var m = JsonConvert.DeserializeObject<ImportMessage>(result.Message);
+
+                            if (m.Release?.Id == null || m.Release?.Publication?.Id == null || m.MetaFileName == null)
+                            {
+                                throw new NullReferenceException();
```

Comment:
Not entirely sure what the motivation here is. Does the `JSONReaderException` not give you enough information when it's thrown?

Reply:
JSONReaderException doesn't get thrown if `result.Message == ""`